dspace/config/dspace.cfg

```diff
@@ -1174,18 +1174,6 @@ plugin.named.org.dspace.content.license.LicenseArgumentFormatter = \
 
 # enable syndication feeds - links display on community and collection home pages
 webui.feed.enable = true
```

Comment:
I believe this can also be removed, as you noted in previous comments that we only need `websvc.opensearch.enable=true` now.

Reply:
Removed this line from the dspace.cfg made sure it wasn't in use anywhere else.